onfido-java/src/main/models/document.json

```diff
@@ -59,6 +59,10 @@
       "type": "boolean",
       "description": "When true the submitted image will undergo an image quality validation",
       "writeOnly": true
+    },
```

Comment:
Is this json shared for request body and response body? Because for documents the location is only available in the request body and not in the response.

Reply:
Good point, that depends on `writeOnly` or `readOnly` flags:
```
    isSentInRequests = !json.readOnly;
    isSentInResponses = !json.writeOnly;
```
https://github.com/onfido/onfido-java/blob/master/generate-models-maven-plugin/src/main/java/com/onfido/models/Property.java#L35-L36

I'm going to fix PR accordingly.